app/crate/genre_taxonomy.py

```
@@ -0,0 +1,855 @@
+                """
+                SELECT
+                    source.slug AS source_slug,
+                    target.slug AS target_slug,
+                    edge.relation_type
+                FROM genre_taxonomy_edges edge
+                JOIN genre_taxonomy_nodes source ON source.id = edge.source_genre_id
+                JOIN genre_taxonomy_nodes target ON target.id = edge.target_genre_id
+                """
+            )
+            edge_rows = cur.fetchall()
+    except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)